server/engine/src/main/antlr4/org/eclipse/lsp/db2/parser/Db2SqlParser.g4

```diff
@@ -38,6 +36,18 @@ binary_host_variable_varbinary_size: T=dbs_integerliteral_expanded {validateInte
 
 host_variable_usage: (USAGE IS?)? SQL TYPE IS;
 
+sql_host_variables: dbs_level_01 entry_name host_variable_usage (result_set_locator | lobWithSize | lobNoSize);
+
+result_set_locator : RESULT_SET_LOCATOR VARYING;
+
+lobWithSize
+    : (BINARY LARGE OBJECT | BLOB | CHARACTER LARGE OBJECT | CHAR LARGE OBJECT | CLOB | DBCLOB) LPARENCHAR dbs_integer k_m_g? RPARENCHAR
```

Comment:
@ishche (question/ doubt) would it help if "BINARY LARGE OBJECT" is combined as one rule , from performance perspective ?